sw/device/silicon_creator/rom/e2e/BUILD

```diff
@@ -1990,61 +1990,144 @@ test_suite(
     ) for lc_state in structs.to_dict(CONST.LCV) for redact in REDACT],
 )
 
-[opentitan_multislot_flash_binary(
-    name = "boot_policy_valid_img_a_{}_b_{}".format(
-        "empty" if offset == "0x0" else "bad",
-        "empty" if offset != "0x0" else "bad",
-    ),
+[
+    opentitan_multislot_flash_binary(
+        name = "sigverify_always_img_a_{}_b_{}".format(
+            "nothing" if slot == "b" else "bad",
+            "nothing" if slot == "a" else "bad",
+        ),
+        srcs = {
+            ":empty_test_slot_{}_corrupted".format(slot): {
+                "key": "test_key_0",
+                "offset": offset,
+            },
+        },
+        devices = ["fpga_cw310"],
+    )
+    for slot, offset in SLOTS.items()
+]
+
+opentitan_multislot_flash_binary(
+    name = "sigverify_always_img_a_bad_b_bad",
     srcs = {
-        ":empty_test_slot_{}_corrupted".format(slot): {
+        ":empty_test_slot_a_corrupted": {
             "key": "test_key_0",
-            "offset": offset,
+            "offset": SLOTS["a"],
+        },
+        ":empty_test_slot_b_corrupted": {
+            "key": "test_key_0",
+            "offset": SLOTS["b"],
+        },
+    },
+    devices = ["fpga_cw310"],
+)
+
+# Since we cannot feed the `assemble_flash_image` rule that is instantiated by
+# the `opentitan_multislot_flash_binary` macro an empty dictionary, we create
+# two images with "nothing" in them by created files of all ones, and stitching
+# them together.
+[genrule(
+    name = "sigverify_always_img_{}_nothing_{}_bin_signed_{}".format(slot, device, key),
+    outs = ["sigverify_always_img_all_ones_{}_bin_signed_{}".format(slot, device, key)],
+    cmd_bash = "dd if=/dev/zero bs=1024 count=1 status=none | tr \'\\0\' \'\\377\' | dd of=$(OUTS) bs=1024 status=none",
+) for slot in SLOTS for device in ["fpga_cw310"] for key in ["test_key_0"]]
+
+opentitan_multislot_flash_binary(
+    name = "sigverify_always_img_a_nothing_b_nothing",
+    srcs = {
+        ":sigverify_always_img_a_nothing": {
+            "key": "test_key_0",
+            "offset": SLOTS["a"],
+        },
+        ":sigverify_always_img_b_nothing": {
+            "key": "test_key_0",
+            "offset": SLOTS["b"],
         },
     },
     devices = ["fpga_cw310"],
-) for slot, offset in SLOTS.items()]
+)
```

Comment:
Instead of all of this, we can simply create an empty file, e.g. by `touch`ing. Bootstrap erases the entire flash before attempting to write anything. I'm not sure if `opentitantool` will complain though. Can you give it a try?

Reply:
oh duhhh yep thats way easier hehe... I just assumed it would complain. but it doesn't! nice, thanks! fixed.